consensus/consensus-types/src/proposal_ext.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright © Aptos Foundation
+
+use crate::common::{Author, Payload, Round};
+use aptos_types::system_txn::SystemTransaction;
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize, Clone, Debug, PartialEq, Eq)]
+pub enum ProposalExt {
```

Comment:
Somewhere validators need to verify the ProposalExt? I thought we hard limit the number of system txns

Reply:
We have inherited `Proposal` verification in `RoundManager::process_proposal`.

The extra system transactions are automatically respecting txn limit/block size limit (if we don't introduce new sys-txn block limit).

Finally in the execution we verify the quorum-certified update (that every sys txn should have).

Anything missing?